user/manager/manager.go

```diff
@@ -120,6 +120,23 @@ func (m *UserManager) Disable(userID string, disabled bool) error {
 	return nil
 }
 
+func (m *UserManager) SetDisplayName(usr user.User, displayName string) error {
+	tx, err := m.begin()
+
+	usr.DisplayName = displayName
+	if err = m.userRepo.Update(tx, usr); err != nil {
+		rollback(tx)
```

Comment:
Generally, you can just do `defer rollback(tx)` after creating the tx. `rollback` is no-op once `tx.Commit` has been successfully called.